### PR TITLE
add json to file function

### PR DESCRIPTION
**Pull request by iamsoorena**
this function will write a json object to a file beautifully.

### CODE REVIEW

**Comment by Chalarangelo** [comment]
Thanks for the contribution! However, I am not exactly sure where we stand on Node.js snippets right now. So far all we have added is browser-based and we haven't really discussed this in-depth. I'm gonna put this PR `on hold` and open an issue to discuss this with the community.

**Comment by Chalarangelo** [comment]
Ignore the last label, it was added by mistake. I'll merge this in a bit and categorize it properly.

File: snippets/json-cleaner.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+### Clean Json objects
```
Comment (Chalarangelo) [inline snippets/json-cleaner.md]: This file is on a different PR, please remove it from this one and we will figure it out in #125.

File: snippets/write-json-to-file.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+### Write a JSON to a file
+
+Write a `json` object to a `.json` file.
+```js
+const fs = require('fs')
+
```
Comment (Chalarangelo) [inline snippets/write-json-to-file.md]: I like this snippet. I can improve the description, but the code is solid. Remove the other file and I will merge this one!

Comment (iamsoorena) [reply snippets/write-json-to-file.md]: ok I will make things ok ASAP

**Pull request merged**